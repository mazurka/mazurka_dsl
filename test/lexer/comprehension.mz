[Var | Var <- Vars]

[mod:fn(Var1) | Var1 <- OtherVars]