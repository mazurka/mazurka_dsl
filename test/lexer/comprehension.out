{list_begin,1}.
{variable,1,'Var'}.
{comp_sep,1}.
{variable,1,'Var'}.
{assign,1}.
{variable,1,'Vars'}.
{list_end,1}.
{list_begin,3}.
{call_begin,3,{mod,fn}}.
{variable,3,'Var1'}.
{call_or_affordance_end,3}.
{comp_sep,3}.
{variable,3,'Var1'}.
{assign,3}.
{variable,3,'OtherVars'}.
{list_end,3}.